adapter/config/mgwtypes.go

```diff
@@ -62,21 +62,21 @@ func NewReceiver() chan string {
 // Config represents the adapter configuration.
 // It is created directly from the configuration toml file.
 type Config struct {
-
```

Comment:
As discussed during the websocket code review when we are renaming the files and DTOs please rename this file to `types.go` we don't need mgw prefix